api/.env.example

```diff
@@ -0,0 +1,7 @@
+JIRA_EMAIL = ""
+JIRA_API_KEY = ""
```

Comment:
While adding this file, I've suggested updating the names of the two Jira environment variables, to make this clearer and more future-proof (as we might want to add more email/API key variables for other services).

Reply:
Makes sense.